RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -950,7 +963,9 @@ void Event::createPixelQuintuplets() {
 
 void Event::addMiniDoubletsToEventExplicit() {
   auto nMDsCPU_buf = allocBufWrapper<unsigned int>(cms::alpakatools::host(), nLowerModules_, queue_);
-  alpaka::memcpy(queue_, nMDsCPU_buf, miniDoubletsBuffers_->nMDs_buf, nLowerModules_);
+  auto nMDs_view =
+      alpaka::createView(devAcc_, miniDoubletsDC_->const_view<MiniDoubletsOccupancySoA>().nMDs(), nLowerModules_);
```

Comment:
unlike in other places, here the "size" is manually set different from the full `nMDs` size `+1`; perhaps a comment can be useful (I recall being confused by this mismatch)